content/livesync/quickstart.textile

```diff
@@ -1,7 +1,228 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+LiveSync facilitates broadcasting realtime updates from backend databases to frontend clients at scale. Leveraging Ably's Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably and in order to all connected clients in realtime.
+
+This quickstart will provide a brief introduction into the LiveSync components integrated into a realtime web application enabling people to comment on a post. Following this, you'll follow instructions to get this demo application running locally.
+
+h2(#breakdown). Understanding the implementation
+
+Each component used within this section is based on the "working demo application":https://github.com/ably-labs/models/examples/livecomments that you'll have running locally by the end of this quickstart.
+
+h3(#model). The model
+
+The @model@ is a data model represntation of a specific part of your frontend application. In the @livecomments@ example, the "model":https://github.com/ably-labs/models/blob/main/examples/livecomments/lib/models/hook.ts#L25-L55 is a specific post, defined by its unique post ID, @post:${id}@.
+
+```[javascript]
+  const modelsClient = new ModelsClient({
+    ably,
+    logLevel: 'trace',
+    optimisticEventOptions: { timeout: 5000 },
+    syncOptions: { retryStrategy: backoffRetryStrategy(2, 125, -1, 1000) },
+  });
+
+  const model = modelsClient.models.get({
+    name: `post:${id}`,
+    channelName: `${channelNamespace}post:${id}`,
+    sync: async () => getPost(id),
+    merge,
+  });
+```
+
+h4(#sync). sync
```

Comment:
I think there's an extra line break in the preview

Reply:
I think it's the styling, There's a larger gap between a header and the following text.